inc/dbmysql.class.php

```diff
@@ -243,16 +222,13 @@ function prepare($query) {
    }
 
    /**
-    * Give result from a mysql result
-    *
-    * @param $result    MySQL result handler
-    * @param $i         Row to give
-    * @param $field     Field to give
-    *
-    * @return Value of the Row $i and the Field $field of the Mysql $result
-   **/
+    * Give result from a sql result
```

Comment:
Most of the time, blocks into doblocks are separated with a blank line (description -> blank line -> parameters -> blank line -> return).
I don't know if this is something required with générators, but I thinks it is more easy to read :)